client/src/API/Constants.ts

```diff
@@ -0,0 +1,11 @@
+export const URL = "http://localhost:5000";
```

Comment:
Can you make the first C in 'Constants' lower-case since it isn't a react component?  Also, pull the constants file into the SRC folder because we have things defined here that aren't specifically for API any more.